docs/glossary.md

```diff
@@ -17,45 +17,65 @@ AWS
     [Amazon Web Services](https://aws.amazon.com/) offers reliable, scalable, and inexpensive cloud computing services.
     Free to join, pay only for what you use.
 
-Linode
-    [Linode.com](https://www.linode.com/) is an American privately owned virtual private server provider company based in Galloway, New Jersey, United States.
-
-DigitalOcean
-    [DigitalOcean, Inc.](https://www.digitalocean.com/) is an American cloud infrastructure provider headquartered in New York City with data centers worldwide.
-
-ZODB
-    [A native object database for Python](https://zodb.org/en/latest/).
-
 Barceloneta
     The default theme for Plone 5.
 
 CMS
     Content Management System
 
+cookiecutter
```

Comment:
In this context, "Cookiecutter" is a proper noun because it is a package name and must be capitalized. However when we use or create a "cookiecutter", then it is not capitalized.

```suggestion
Cookiecutter
```